src/components/FilterGroup.tsx

```diff
@@ -75,11 +83,17 @@ export function FilterGroup({
     };
   }, [customCssClasses]);
 
+  const collapsibleLabelCssClasses = useMemo((): CollapsibleLabelCssClasses => {
+    return {
+      titleLabel: cssClasses.titleLabel
+    };
+  }, [cssClasses]);
+
   function renderTitle() {
     return collapsible
-      ? <CollapsibleLabel label={title} />
+      ? <CollapsibleLabel label={title} customCssClasses={collapsibleLabelCssClasses} />
       : (title &&
-        <div className='text-neutral-dark text-sm font-medium text-left mb-4'>
+        <div className={twMerge('mb-4', builtInCssClasses.titleLabel, cssClasses.titleLabel)}>
```

Comment:
why is there an extra `mb-4` class here that isn't in the CollapsibleLabel styling? can it be removed so that they are the same?

Reply:
it was off by one class because `mb-4` is on a wrapping class on `CollapsibleLabel`, that the non-collapsible version doesn't have